transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java

```diff
@@ -94,6 +97,10 @@ public LoggingFilter(Logger logger) {
         this(logger, "%s");
     }
 
+    public LoggingFilter(Logger logger, IoSession session, String transportName) {
```

Comment:
It seems like this restricts a logging filter to be used per session. Previously, a logging filter could be used across all sessions (though, I didn't find any usages for that). One instance of the filter was *not* shared across the sessions in the current code, so should be fine.